hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -378,6 +390,7 @@ public Response differentialPush(@Context HttpServletRequest request,
         GrailParams params = new GrailParams();
         params.setUser(user);
         params.setPushUrl(RAILSPORT_PUSH_URL);
+        params.setComment(params.getComment());
```

Comment:
should be reqParams.getComment().
Also should you also be setting the hashtags and the source here as well?